### PR TITLE
Improved integer looping. Silenced warning.

### PR DESCRIPTION
**Pull request by elland**

### CODE REVIEW

**Comment by designatednerd** [comment]
Looks like you might wanna update the Travis settings: 

screen shot 2018-02-06 at 9 27 07 am

File: Sources/Integer+Repetition.swift [hunk 1]
```diff
@@ -8,4 +8,12 @@ public extension Int {
...
         }
     }
+
+    public func times(_ closure: (_ i: Int) -> ()) {
+        if self > 0 {
+            for i in 0..<self {
+                closure(i)
+            }
+        }
+    }
```
Comment (designatednerd) [inline Sources/Integer+Repetition.swift]: worth writing a test or two on this?

File: Sources/Integer+Repetition.swift [hunk 2]
```diff
@@ -8,4 +8,12 @@ public extension Int {
             }
         }
     }
+
+    public func times(_ closure: (_ i: Int) -> ()) {
+        if self > 0 {
```
Comment (designatednerd) [inline Sources/Integer+Repetition.swift]: Why not `guard self >0 else {`... ?

**Comment by elland** [comment]
Or move it to Circle.

**Comment by elland** [comment]
@3lvis I think you're needed, to move it to CircleCI?

File: Tests/Tests.swift [hunk 3]
```diff
@@ -42,4 +42,21 @@ class Tests: XCTestCase {
...
+        var count = 0
+        var accumulator = 0
+
+        20.times {
+            count += 1
+        }
+
+        // i is 0-indexed.
+        3.times { i in
+            accumulator += (i+1)
```
Comment (designatednerd) [inline Tests/Tests.swift]: formatting nitpick: Can you please put spaces around the `+`

**Comment by 3lvis** [comment]
CircleCI all the things!

**Comment by 3lvis** [comment]
CircleCI is working on master now.

**Comment by 3lvis** [comment]
Rebased and CircleCI is working on this branch now 🎉

**Review by designatednerd — APPROVED** [review]
once those spaces are added, :shipit:

**Comment by elland** [comment]
@3lvis 🙌

**Comment by elland** [comment]
@designatednerd 🚀

**Comment by designatednerd** [comment]
@Elland :shipit:

**Pull request merged**